src/arrayMethodReduce.js

```diff
@@ -4,8 +4,20 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, props) {
```

Comment:
`props` is not perfect name for the variable
The meaning of it is to pass the initial value to accumulator - you need to pick something descriptive